### PR TITLE
[TELCODOCS-488] - OCP 4.10 release note known issues for RAN GA

### PR DESCRIPTION
**Pull request by aireilly**
Fixes https://issues.redhat.com/browse/TELCODOCS-488, Merge to enterprise-4.10

Preview: https://deploy-preview-41683--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-ran-ga-known-issues

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 // TODO: ^^^ Add BZ when created.
 
+[id="ocp-4-10-ran-ga-known-issues"]
+* The Kubelet service monitor scrape interval is currently set to a hard-coded value. This means that there is less available CPU resources for workloads.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2548]: @kenyis Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2035046[*BZ#2035046*])
+
+* Deploying a single node Openshift cluster for vRAN Distributed Units can take up to 4 hours.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2551]: @kenyis Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2035036[*BZ#2035036*])
+
+* Currently, you can add a `managedPolicy` in a `ClusterGroupUpgrade` manifest that is already enforced. This should not be possible.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2554]: @alosadagrande Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2044304[*BZ#2044304*])
+
+* If a `ClusterGroupUpgrade` CR has a `blockingCR` specified, and that `blockingCR` fails silently, for example if there is a typo in the list of clusters, the `ClusterGroupUpgrade` CR is applied even though the `blockingCR` is not applied in the cluster.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2557]: @yliu127 Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042601[*BZ#2042601*])
+
+* If a `ClusterGroupUpgrade` CR validation fails for any reason, for example, because of an invalid spoke name, no status is available for the `ClusterGroupUpgrade` CR, as if the CR is disabled.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2560]: @yliu127 Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 6]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040828[*BZ#2040828*])
+
+* Currently, for `ClusterGroupUpgrade` CR state changes, there is only a single condition type available - `Ready`. The `Ready` condition can have a status of `True` or `False` only. This does not refect the range of states the `ClusterGroupUpgrade` CR can be in.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2563]: @yliu127 Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 7]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2057209[*BZ#2057209*])
+
+* In the default ZTP Argo CD configuration, cluster names cannot begin with `ztp`. Using names starting with `ztp` for clusters deployed with Zero Touch Provisioning (ZTP) results in provisioning not completing. As a workaround, ensure that either cluster names do not start with `ztp`, or adjust the Argo CD policy application namespace to a pattern that excludes the names of your clusters but still matches your policy namespace. For example, if your cluster names start with `ztp`, change the pattern in the Argo CD policy app configuration to something different, like `ztp-`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2049154[*BZ#2049154*])
+
+* During a spoke cluster upgrade, one or more reconcile errors is recorded in the container log. The number of errors corresponds to the number of child policies. The error causes no noticeable impact to the cluster. The following is an example of the reconcile error:
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2648]: @yliu127 Please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 8]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042601[*BZ#2042601*])
+
+* If a `ClusterGroupUpgrade` CR validation fails for any reason, for example, because of an invalid spoke name, no status is available for the `ClusterGroupUpgrade` CR, as if the CR is disabled.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040828[*BZ#2040828*])
+
+* Currently, for `ClusterGroupUpgrade` CR state changes, there is only a single condition type available - `Ready`. The `Ready` condition can have a status of `True` or `False` only. This does not refect the range of states the `ClusterGroupUpgrade` CR can be in.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042596[*BZ#2042596*])
+
+* The alpha channel for the PTP Operator reports an error that prevents a `PtpConfig` CR from being applied when the `summary_interval` value does not match the value set for `logSyncInterval`. To work around this issue, update all reference PTP configurations to set `summary_interval` and `logSyncInterval` to -4.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2566]: @imiller0 review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 9]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+[source,yaml]
+----
+spec:
+  approval: Manual
+  approved: true
+  clusterServiceVersionNames:
+  - openshift-gitops-operator.v1.3.2
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046775[*BZ#2046775*])
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2622]: @mcornea review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 10]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+
+* Sometimes, when provisioning spoke clusters, argoCD applications with 100 or more `Siteconfig` CRs can cause `OOMKilled` memory errors on the hub cluster, even when the `openshift-gitops-application-controller-0` is allocated 40GB memory.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2026476[*BZ#2026476*])
+
+* When deploying a multi-node cluster on bare-metal nodes, the machine config pool (MCP) adds an additional CRI-O drop-in that circumvents the link:https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/README.md#container-mount-namespace[container mount namespace drop-in]. This results in CRI-O being in the base namespace while the kubelet is in the hidden namespace. All containers fail to get any kubelet-mounted filesystems, such as secrets and tokens.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2028590[*BZ#2028590*])
+
+* Installing {rh-rhacm} 2.4.2 on a bare-metal host causes the Kubernetes API Server operator to report the following error:
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2630]: @mcornea review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 11]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
++
+[source,terminal]
+----
+MutatingAdmissionWebhookConfigurationDegraded: managedclustermutators.admission.cluster.open-cluster-management.io: x509: certificate signed by unknown authority
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2034913[*BZ#2034913*])
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2637]: @mcornea review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 12]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+spec:
+  approval: Manual
+  approved: true
+  clusterServiceVersionNames:
+  - openshift-gitops-operator.v1.3.2
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046775[*BZ#2046775*])
+
+* Sometimes, when provisioning spoke clusters, argoCD applications with 100 or more `Siteconfig` CRs can cause `OOMKilled` memory errors on the hub cluster, even when the `openshift-gitops-application-controller-0` is allocated 40GB memory.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2624]: @hanqiuzh review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 13]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2026476[*BZ#2026476*])
+
+* When deploying a multi-node cluster on bare-metal nodes, the machine config pool (MCP) adds an additional CRI-O drop-in that circumvents the link:https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/README.md#container-mount-namespace[container mount namespace drop-in]. This results in CRI-O being in the base namespace while the kubelet is in the hidden namespace. All containers fail to get any kubelet-mounted filesystems, such as secrets and tokens.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2627]: @lack review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 14]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
++
+[source,terminal]
+----
+MutatingAdmissionWebhookConfigurationDegraded: managedclustermutators.admission.cluster.open-cluster-management.io: x509: certificate signed by unknown authority
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2034913[*BZ#2034913*])
+
+* Installing {rh-rhacm} 2.5.0 in the `ocm` namespace causes the `infrastructure-operator` pod to fail due to insufficient privileges.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046554[*BZ#2046554*])
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2640]: Thuy Nguyen  review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 15]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+
+* {product-title} limits object names to 63 characters. If a policy name defined in a `PolicyGenTemplate` CR approaches this limit the Topology Aware Lifecycle Operator cannot create child policies. When this occurs, the parent policy will remain in a `NonCompliant` state.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2057209[*BZ#2057209*])
+
+* In the default ZTP Argo CD configuration, cluster names cannot begin with `ztp`. Using names starting with `ztp` for clusters deployed with Zero Touch Provisioning (ZTP) results in provisioning not completing. As a workaround, ensure that either cluster names do not start with `ztp`, or adjust the Argo CD policy application namespace to a pattern that excludes the names of your clusters but still matches your policy namespace. For example, if your cluster names start with `ztp`, change the pattern in the Argo CD policy app configuration to something different, like `ztp-`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2049154[*BZ#2049154*])
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2646]: Joshua Clark please review/LGTM - RAN GA known issue.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 16]
```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
...
+sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
+        /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.2/pkg/internal/controller/controller.go:253
+sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
+        /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.2/pkg/internal/controller/controller.go:214
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2043301[*BZ#2043301*])
+
+* In {product-title} 4.10, `linxptp` daemon usage has more than doubled since earlier measurements in version 4.9. For an ordinary clock PTP configuration using the Intel e810 NIC, CPU usage approaches 45 mc. No faults are recorded in the logs. These results are found with the following NIC configuration:
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:2662]: @browsell  Please review/LGTM - RAN GA known issue.

**Review by aireilly — COMMENTED** [review]
Adding RAN GA known issue reviewers

File: release_notes/ocp-4-10-release-notes.adoc (hunk 14 above)
Comment (thuyn-581) [reply release_notes/ocp-4-10-release-notes.adoc:2640]: LGTM

**Review by slovern — COMMENTED** [review]
Added some comments for corrections.
One questrion outstanding: The list of "Known issues in OCP 4.10.x RAN GA" includes
https://bugzilla.redhat.com/show_bug.cgi?id=2037874
However there is no documentation added for this BZ. 
I've contacted colleagues and am waiting for clarification on whether this was intentional or not. Will confirm if an entry is needed to account for this BZ. UPDATE: received confirmation from @kenyis that no documentation is needed here for this bug.

**Comment by aireilly** [comment]
Closing this. This work will be completed in https://github.com/openshift/openshift-docs/pull/44003

File: release_notes/ocp-4-10-release-notes.adoc (hunk 8 above)
Comment (imiller0) [reply release_notes/ocp-4-10-release-notes.adoc:2566]: lgtm

File: release_notes/ocp-4-10-release-notes.adoc (hunk 12 above)
Comment (serngawy) [reply release_notes/ocp-4-10-release-notes.adoc:2624]: The issue doesn't not exist but the siteconfig & ArgoCD still shipped with RAN 4.10

File: release_notes/ocp-4-10-release-notes.adoc (hunk 4 above)
Comment (yliu127) [reply release_notes/ocp-4-10-release-notes.adoc:2557]: LGTM

File: release_notes/ocp-4-10-release-notes.adoc (hunk 5 above)
Comment (yliu127) [reply release_notes/ocp-4-10-release-notes.adoc:2560]: LGTM

File: release_notes/ocp-4-10-release-notes.adoc (hunk 6 above)
Comment (yliu127) [reply release_notes/ocp-4-10-release-notes.adoc:2563]: LGTM

File: release_notes/ocp-4-10-release-notes.adoc (hunk 7 above)
Comment (yliu127) [reply release_notes/ocp-4-10-release-notes.adoc:2648]: LGTM

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (kenyis) [reply release_notes/ocp-4-10-release-notes.adoc:2548]: lgtm

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (kenyis) [reply release_notes/ocp-4-10-release-notes.adoc:2551]: lgtm

File: release_notes/ocp-4-10-release-notes.adoc (hunk 13 above)
Comment (lack) [reply release_notes/ocp-4-10-release-notes.adoc:2627]: /lgtm

Comment (lack) [reply release_notes/ocp-4-10-release-notes.adoc:2627]: Also commented in #44003

File: release_notes/ocp-4-10-release-notes.adoc (hunk 15 above)
Comment (josclark42) [reply release_notes/ocp-4-10-release-notes.adoc:2646]: /LGTM

**Pull request closed without merging**